### PR TITLE
OBS_GetMCDOMVT can receive several categories

### PR DESCRIPTION
**Pull request by antoniocarlon**
Added multi category support for the `OBS_GetMCDOMVT` function

Closes https://github.com/CartoDB/bigmetadata/issues/516
Closes https://github.com/CartoDB/observatory-extension/issues/339
Closes https://github.com/CartoDB/bigmetadata/issues/518
Closes https://github.com/CartoDB/observatory-extension/issues/341

### CODE REVIEW

**Comment by antoniocarlon** [comment]
The performance is awful, I'm taking a look at it

**Comment by antoniocarlon** [comment]
Added multi-month support to the function too

File: src/pg/sql/45_observatory_mvt.sql [hunk 1]
```diff
@@ -263,17 +263,79 @@ BEGIN
 END
 $$ LANGUAGE plpgsql PARALLEL RESTRICTED;
 
+CREATE OR REPLACE FUNCTION cdb_observatory.OBS_DecodeMCCategory(category TEXT)
```
Comment (ethervoid) [inline src/pg/sql/45_observatory_mvt.sql]: Maybe is a bit hardcore to create a function to return this. What do you think?

Comment (antoniocarlon) [reply src/pg/sql/45_observatory_mvt.sql]: Totally. I'm removing it :+1:

File: src/pg/sql/45_observatory_mvt.sql [hunk 2]
```diff
@@ -263,17 +263,66 @@ BEGIN
 END
 $$ LANGUAGE plpgsql PARALLEL RESTRICTED;
 
+CREATE OR REPLACE FUNCTION cdb_observatory.OBS_GetMCDates(
```
Comment (ethervoid) [inline src/pg/sql/45_observatory_mvt.sql]: Are you using this function?

Comment (ethervoid) [reply src/pg/sql/45_observatory_mvt.sql]: Or is a public function to be used to pass the months to the obs_getmcdo... function?

Comment (antoniocarlon) [reply src/pg/sql/45_observatory_mvt.sql]: Is a helper function that can be used to get the months needed by the `obs_getmcdomvt` function

**Pull request merged**